git_wrapper/tag.py

```diff
@@ -72,3 +72,16 @@ def push(self, name, remote, dry_run=False):
                 self.git_repo.git.push(remote, name)
         except git.GitCommandError as ex:
             raise_from(exceptions.PushException(msg), ex)
+
+    def list_tags(self):
+        """List git tags in the repository.
+        """
+
+        try:
+            tags_list = self.git_repo.repo.tags
+        except git.GitCommandError as ex:
+            tags_list = []
+            msg = "Error listing tags for {repo}. Error: {error}".format(error=ex, repo=self.git_repo)
```

Comment:
self.git_repo is an object so it doesn't work well as an error message, you end up with output like this:

git_wrapper.exceptions.TaggingException: Error listing tags for <git_wrapper.repo.GitRepo object at 0x7f11703b9fd0>. Error: 

I'm not sure if it's absolutely necessary to have the repo name in this, since the caller should know what they initialised GitRepo with, but if it's there, it should be a human-readable string.